blivet/static_data/lvm_info.py

```diff
@@ -37,7 +37,13 @@ class LVsInfo(object):
     @property
     def cache(self):
         if self._lvs_cache is None:
-            lvs = blockdev.lvm.lvs()
+            try:
+                lvs = blockdev.lvm.lvs()
+            except blockdev.BlockDevNotImplementedError:  # pylint: disable=catching-non-exception
```

Comment:
It should be possible to catch ``NotImplementedError`` here too (``BlockDevNotImplementedError`` should be inheriting from it)